### PR TITLE
Update MFS.groovy declaring the map as output

### PR DESCRIPTION
**Pull request by dennis-behm**
This is addressing #36, #346 and incorporates feedback from the field.

What is changed:
* The MFS source is defined as a build output in the DBB Build report. (Because the process generated environment specific formats, and the number for formats cannot be determined).
* The phase2 process becomes optional.

Commits need to be squash (started from `main` instead of `develop`)

### CODE REVIEW

File: languages/MFS.groovy [hunk 1]
```diff
@@ -63,6 +73,8 @@ sortedList.each { buildFile ->
 			props.error = "true"
 			buildUtils.updateBuildResult(errorMsg:errorMsg,logs:["${member}.log":logFile])
 		}
+		
+		}
```
Comment (M-DLB) [inline languages/MFS.groovy:77]: For better readability, can you change the indentation of the block between L65 and L75?
Thank you!

**Review by M-DLB — COMMENTED** [review]
Looks good, just a small change for the indentation.

File: languages/MFS.groovy (hunk 1 above)
Comment (dennis-behm) [reply languages/MFS.groovy:74]: @M-DLB Applied pretty print.

**Review by M-DLB — APPROVED** [review]
Thank you!

**Pull request merged**